roles/bindle-profiles/vars/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 # The git repo branches, if you're working in production then use the latest stable tagged version
-bindle_git_branch: "2.0-alpha.4"
-seqware_bag_git_branch: "1.0-beta.0"
```

Comment:
Make sure to revert this to valid versions before closing this pull request. 
For testing purposes, you can override these variables from the CLI http://docs.ansible.com/playbooks_variables.html#passing-variables-on-the-command-line